cppclient/tests/api_json.cc

```diff
@@ -62,5 +62,38 @@ TEST_F(IotaJsonAPITest, GetBalances) {
   EXPECT_EQ(api.request, req);
   EXPECT_EQ(response, expected);
 }
+TEST_F(IotaJsonAPITest, GetNodeInfo) {
```

Comment:
There should be one blank line to split `GetNodeInfo` item and the previous one.